packages/hash/frontend/src/blocks/page/collab/EditorConnection.ts

```diff
@@ -341,7 +341,7 @@ export class EditorConnection {
             type: "update",
             transaction: tr,
             requestDone: true,
-            version: data.version ?? this.state.version,
+            version: data.version || this.state.version,
```

Comment:
This seems incorrect. If `data.version` is 0, it's still supposed to overwrite our version. 

If it's complaining because `version` is always there, we should either just removing the `||` or correct the typing if that isn't correct. 

This doesn't really matter in reality because version should never go backwards, so should never be 0 at this point.